.changes/unreleased/FEATURES-20231006-135721.yaml

```diff
@@ -0,0 +1,6 @@
+kind: FEATURES
```

Comment:
Let's also add an additional changie entry of kind `FEATURES` for the new `ListDatabaseAuthenticationConfigs()` endpoint

> FEATURES: These are major new improvements that deserve a special highlight, such as a new API endpoint or service.
https://github.com/bastionzero/bastionzero-sdk-go/blob/master/CONTRIBUTING.md#changelog

Reply:
b574bc2d38a8307385ccf82081f6a1253b339e7c